nextflow_schema.json

```diff
@@ -10,22 +10,26 @@
             "type": "object",
             "fa_icon": "fas fa-terminal",
             "description": "Define where the pipeline should find input data and save output data.",
-            "required": [
-                "input"
-            ],
             "properties": {
-                "input": {
+                "reads": {
```

Comment:
```suggestion
                "input": {
```

Reply:
I remember it used to be `--reads` but it was changed to `--input` in a previous template sync in an effort to standardize parameter names across pipelines.